### ISSUE TITLE
textarea field type still doing Input Sanitisation

### ISSUE THREAD

**Issue by vbeskrovny** [bolt/core#2248]
Changed the contenttypes.yaml content value for the homepage from 'redactor' to 'textarea'. Pasted '<style></style>'. Saved. And it was sanitised...

Details
-------

| Question                      | Answer
|----------------- | ------
| Relevant Bolt Version | 4.1.8
| Install type                   |  Composer install
| BC Break                     | yes/no ???
| PHP version                 | 7.3
| Web server                  | Nginx
| For UX/UI issues        | FF 83

Reproduction
------------

If you're filing a bug, please describe how to reproduce it. Include as much
relevant information as possible, such as:

**Comment by I-Valchev** [comment]
Yes... it _should_ be sanitised.

You can turn it off by adding `sanitise: false` to the field options.

**Comment by vbeskrovny** [comment]
> 
> 
> Yes... it _should_ be sanitised.
> 
> You can turn it off by adding `sanitise: false` to the field options.
> 
> Does it work then? :-)

Thank you for the feedback.
Do you mean I have to make it like that?
![image](https://user-images.githubusercontent.com/38423084/102108530-58dc0300-3e33-11eb-9826-f46f8d7e8a0a.png)

**Comment by vbeskrovny** [comment]
I did the following:

![image](https://user-images.githubusercontent.com/38423084/102109495-65ad2680-3e34-11eb-8765-dc99b3a3fae0.png)

And nothing works :)

**Comment by I-Valchev** [comment]
Hi @vbeskrovny , sorry for the ambiguity, the correct definition is like so:
```yaml
...
    content:
        type: textarea
        sanitise: false
...
```

I put the following in a textarea field using Bolt 4.1.9:

![image](https://user-images.githubusercontent.com/7093518/102213571-37ccee00-3ed7-11eb-8451-d2c52a6feeb8.png)

When I go to the frontend on that page, I see the `hi` alert and the background is red (as per the style).

I'd say update to 4.1.9, and set the definition correctly. Then make sure you run `bin/console cache:clear` to clear the cache. It _should_ work :-)

**Comment by vbeskrovny** [comment]
No need to sorry. I am not 100% familiar with the system and thanks for the feedback.

I did the following:
![image](https://user-images.githubusercontent.com/38423084/102219775-39e77a80-3ee0-11eb-9019-28e73a2f2a50.png)

And it did not work.

Still erased. The next step would be to perform an update (as you suggested) and try again.

Thanks!

**Comment by I-Valchev** [comment]
hi @vbeskrovny , were you able to solve the issue by updating?

**Comment by vbeskrovny** [comment]
Hi, thanks for asking. Nope. I gave up... sorry.

**Comment by napley** [comment]
Hi,

I have the same issue, form and input are remove on save.

It is because the class TextareaField not extends implements RawPersistable

**Comment by I-Valchev** [comment]
@napley can you show an example of how to replicate this, please?

The TextareaField should not implement `RawPersistable` in my view. Rather, that is why we have the `sanitise: false` option, which should be used to not sanitise a field.

**Comment by napley** [comment]
@I-Valchev The commit https://github.com/bolt/core/pull/2198/commits/ace7a65258e0169afd4727a4e3d1efae0d661e5f is not in bolt/core 4.1.10, so the sanitise option doesn't exist.

> The TextareaField should not implement RawPersistable in my view.  

I am not against this, but the documentation say the reverse https://docs.bolt.cm/4.0/fields/textarea#input-sanitisation

**Comment by I-Valchev** [comment]
@napley good spot, I didn't notice this! Thanks for pointing it out, I've opened a PR to get that change in 4.1, meaning it will be fixed in the next maintenance (4.1.x) release of Bolt.

I'm linking this also to an issue in `bolt/docs` so that will get fixed too. Thanks again @napley
